packages/blockchain-wallet-v4-frontend/src/scenes/BuySell/CoinifyCheckout/Content/OrderHistory/template.success.js

```diff
@@ -49,7 +49,16 @@ const OrderHistory = (props) => {
           <Text size='15px' weight={400}>
             <FormattedMessage id='scenes.buysell.coinifycheckout.trades.pending' defaultMessage='Pending Orders' />
           </Text>
-          <OrderHistoryTable trades={filter(isPending, trades)} conversion={conversion} handleFinishTrade={trade => finishTrade(trade)} handleDetailsClick={trade => showModal('CoinifyTradeDetails', { trade })} handleTradeCancel={cancelTrade} status={status} cancelTradeId={cancelTradeId} />
+          <OrderHistoryTable
+            trades={filter(isPending, trades)}
+            conversion={100}
+            handleFinishTrade={trade => finishTrade(trade)}
+            handleDetailsClick={trade => showModal('CoinifyTradeDetails', { trade })}
+            handleTradeCancel={cancelTrade}
+            status={status}
+            canTrade={canTrade.data}
```

Comment:
Never use .data on a Remote. If the data is not necessary, use getOrElse(undefined) and handle the undefined case later on. If the data is necessary, include the Remote (here canTrade) in the data on which we apply the cata in the index.js

Reply:
@sixtedemaupeou good catch, thanks! fixed this.